docs/reference/configurations.md

```diff
@@ -1,3 +1,5 @@
 # Configurations
 
-See [Configure](https://charmhub.io/discourse-k8s/configure).
\ No newline at end of file
+See [Configurations](https://charmhub.io/discourse-k8s/configure).
+
```

Comment:
There is an extra line here. Is this intended?
Since there is no extra line at the same place in `docs/reference/actions.md`. See above comment.